pkg/apis/knative/v1alpha1/deployer_validation_test.go

```diff
@@ -125,3 +125,65 @@ func TestValidateDeployerSpec(t *testing.T) {
 		})
 	}
 }
+
+func TestValidateScale(t *testing.T) {
```

Comment:
let's add one scaling test case that produces an error back to TestValidateDeployerSpec just to confirm that the spec is calling scale.Validate() and reporting the errors under the expected field path.

Reply:
Added a test to the DeployerSpec validation 👍